chrome_manifest_v3/inject.js

```
@@ -2,18 +2,18 @@
-	} else if (
-		typeof window !== "undefined" &&
-		window.opener !== null &&
-		typeof window.opener.GLOBALS !== "undefined"
-	) {
-		userEmail = window.opener.GLOBALS[10];
-	}
-	var G2T_event = new CustomEvent("g2t_connect_extension", {
-		detail: { userEmail: userEmail },
-	});
-	document.dispatchEvent(G2T_event);
+  var userEmail = '?';
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)